test/Dommel.Tests/ParameterPrefixTest.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Dapper;
+using Moq;
+using Moq.Dapper;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Data;
+using System.Reflection;
+using System.Text;
+using Xunit;
+using static Dommel.DommelMapper;
+using System.Data.SqlClient;
+using System.ComponentModel.DataAnnotations;
+
+namespace Dommel.Tests
+{
+    [Collection("Use Dommel Log to check on results")]
```

Comment:
What is the use of this?

Reply:
This is for the tests don't run in parallel. Tests with the same Collection run in sequence.
I've got errors from tests because the log function is static, so the results got altered in parallel.

Reply:
But with this new version I don't get the same errors, I've just ran without it and it is all good. I will remove them.

Reply:
On AppVeyor I've got the tests erro because of the sequence, so I've put the attribute "collection" again.

Reply:
I had to put the attribute "Collection" to the new test "Project" too.

Reply:
Fair enough 👍